oggm-edu/flowline_model.ipynb

```diff
@@ -4,14 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<p>There may be a misunderstanding here:</p><p> </p><pre class="ql-syntax" spellcheck="false"># Retreive the new flowline from the model and plot it. At the moment we only
# have one "future" flowline, but we pick the last one anyway.
current_flowline = model.fls[-1]
</pre><p> </p><p>The model stores a list of flowlines because OGGM allows multiple flowlines for glaciers. the last one is the "main one". I think that for OGGM-Edu it's better to use 0 as index, so I suggest:</p><p> </p><pre class="ql-syntax" spellcheck="false"># Retreive the new flowline from the model and plot it.
# OGGM can have multiple flowlines, here we have only one.
current_flowline = model.fls[0]
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OGGM/oggm-edu-notebooks/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Done fc01bd5